index.js

```diff
@@ -10,16 +10,12 @@ const aliased = [
   'utils',
 ];
 
-function isInternalModule({ moduleName }) {
-  return moduleName.startsWith('@qsi/');
-}
-
 function isAliasedModule({ moduleName }) {
   return aliased.some(n => moduleName.startsWith(n));
 }
 
 function isCssModule({ moduleName }) {
-  return moduleName.includes('@qsi/ui-theme') || moduleName.includes('.css');
```

Comment:
I think it's fine to remove, its not even actually css it's javascript exported here.